jsapp/js/components/transferProjects.module.scss

```diff
@@ -0,0 +1,65 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+
+.root {
+  background: colors.$kobo-white;
+  padding: sizes.$x20;
+  margin: sizes.$x24 sizes.$x40;
+  border: 1px solid colors.$kobo-gray-92;
+  border-radius: sizes.$x5;
+}
+
+.bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  button {
+    max-width: sizes.$x180;
+    height: sizes.$x38;
+  }
+}
+
+.description {
+  margin-right: sizes.$x16;
+}
+
+.copy {
+  margin-top: sizes.$x8;
+}
+
+.modalBody {
+  line-height: sizes.$x24;
+  padding: 0 sizes.$x24;
+
+  p:first-of-type {
+    margin-top: 0;
+  }
```

Comment:
Let's add this above:

```scss
p {
  margin: sizes.$x16 0;
}
```

This most possibly will not change much, but we only have `p {margin-bottom: 16px;}` in base styles file, and `margin-top` comes from browser defaults - and we don't control what this is :)